jdaviz/configs/mosviz/tests/test_data_loading.py

```diff
@@ -24,7 +28,7 @@ def test_load_spectrum1d(mosviz_helper, spectrum1d):
 
     assert isinstance(data[label], Spectrum1D)
 
-    with pytest.raises(TypeError):
+    with pytest.raises(AttributeError):
```

Comment:
Exception class changed because it now crashes in metadata parsing. I don't think it matters?